packages/EasyApiToken/src/External/JwtTrait.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyApiToken\External;
+
+use Firebase\JWT\JWT as BaseJWT;
+use ReflectionMethod;
+
+/**
+ * @deprecated compatibility layer, will be removed in 5.0
+ * @internal
+ */
+trait JwtTrait
```

Comment:
Let's name it a bit more obviously?
Dunno, maybe smth like `FirebaseJwtVersionControlTrait`, or just `FirebaseJwtVersionTrait`?
wdyt?

Reply:
I like `FirebaseJwtVersionTrait`.

Reply:
Done